opm/simulators/wells/SingleWellState.cpp

```diff
@@ -66,6 +67,25 @@ void SingleWellState::open() {
     this->status = Well::Status::OPEN;
 }
 
+
+double SingleWellState::sum_connection_rates(const std::vector<double> connection_rates) const {
```

Comment:
Was the parameter type here supposed to be a reference (`const vector<>&`) or did you in fact mean to create an independent copy of the input object.

Reply:
It should have been const ref - PR added.